### DISCUSSION TITLE
It is difficult to differentiate between 4/1 and 1/1 on timeline in editor

### DISCUSSION THREAD

**Discussion by ChenMerami** [ppy/osu#29516] [Ideas] [answered]
Being able to easily identify 4/1 tick on the timeline is essential for many mapping concepts and I always have to strain my eyes and get close to monitor to be able to do that in lazer. I don't think it is done good in stable since the difference is 10px on a 2px wide line, but it is at least possible to identify 4/1 at a glance.

![Untitled-2](https://github.com/user-attachments/assets/fca36336-8b04-43b2-a215-f0f5fa2305f7)

I don't think making it wider is a good solution since it would then interfere to much with waveforms in timing sections. Making other ticks shorter is a bad option too since it would be difficult to see the tick on which slider is placed.

![image](https://github.com/user-attachments/assets/4dfaf1fa-47a3-43fa-b032-d212ee062d26)

The only good solution I see is to change its shape (maybe make it T shaped or capital I or stick a dot on top doesn't matter)

![image](https://github.com/user-attachments/assets/2d16491d-50ff-4ce6-bc35-61e5219cbec1)

**Comment by peppy** [comment]
I'll take another look into this.

**Reply by peppy** [reply, marked as answer]
Should be much better with https://github.com/ppy/osu/pull/29545.
